java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -7609,16 +7610,6 @@ public void sscan(BaseClient client) {
         assertEquals(initialCursor, result[resultCursorIndex]);
         assertDeepEquals(new String[] {}, result[resultCollectionIndex]);
 
-        // Negative cursor
```

Comment:
Why do you remove that?

Reply:
This got duplicated a few times due to merge issues.